### ISSUE TITLE
请教目标攻击中的logit损失问题

### ISSUE THREAD

**Issue by sevenPot7** [Trustworthy-AI-Group/TransferAttack#67]
目标攻击中的logit损失大多都是如下这样写
```
real = logits.gather(1,label.unsqueeze(1)).squeeze(1)
logit_dists = ( -1 * real)
loss = logit_dists.sum()
```
请问为什么要求sum()? 如果改成mean()会对结果有影响吗？

**Comment by xiaosen-wang** [comment]
The primary distinction between using the `sum` and `mean` functions to compute gradients lies merely in a constant factor. When the `sign` function is employed in the attack, this difference does not impact the overall performance.

**Comment by zqyjww** [comment]
May I ask if I can try to generate adversarial samples using ResNet50 or ResNet152 as proxy models? For example, Python main. py -- input_ir/ path/to/data --output_dir adv_data/logit/resnet18_targeted --attack logit --model=resnet50 --targeted
